e2e-pw/src/oss/specs/dynamic-groups.spec.ts

```diff
@@ -12,35 +12,54 @@ const test = base.extend<{ grid: GridPom; modal: ModalPom }>({
   },
 });
 
-["mp4", "pcd", "png"].forEach((extension) => {
-  const datasetName = getUniqueDatasetNameWithPrefix(
-    `${extension}-dynamic-groups`
-  );
-
-  test.beforeAll(async ({ fiftyoneLoader }) => {
-    await fiftyoneLoader.executePythonCode(`
-    import fiftyone as fo
-    dataset = fo.Dataset("${datasetName}")
-    dataset.persistent = True
-
-    samples = []
-    for i in range(0, 100):
-        sample = fo.Sample(filepath=f"{i}.${extension}", dynamic_group=i % 10)
-        samples.append(sample)
-    
-    dataset.add_samples(samples)
-    view = dataset.group_by("dynamic_group")
-    dataset.save_view("dynamic-group", view)`);
-  });
+// TODO: omitting 'mp4' because https://github.com/voxel51/fiftyone/issues/3421
```

Comment:
Maybe we could change the saved view to include `order_by="filepath"` to avoid omission? Noting that this is a core SDK issue

Reply:
That didn't work, unfortunately / surprisingly :' (